app/javascript/articles/Article.jsx

```diff
@@ -0,0 +1,136 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+import { articlePropTypes } from '../src/components/common-prop-types/article-prop-types';
+import {
+  CommentsCount,
+  ContentTitle,
+  OrganizationHeadline,
+  PublishDate,
+  ReadingTime,
+  SaveButton,
+  SearchSnippet,
+  TagList,
+  ReactionsCount,
+} from './components';
+import { PodcastArticle } from './PodcastArticle';
+
+/* global timeAgo */
+
+export const Article = ({
+  article,
+  currentTag,
+  isBookmarked,
+  reactionsIcon,
+  commentsIcon,
+  videoIcon,
+}) => {
+  if (article && article.type_of === 'podcast_episodes') {
```

Comment:
As mentioned, this is based off of the `buildArticleHTML.js.erb`. It was one big if/else/elseif to build out the required markup. I've broken it up into smaller Preact components, but perhaps in a second phase, we could have the main `<Feed />` component check the type of article and then load, e.g. `<UserArticle />`, '<VideoArticle />` etc.